src/js/containers/award/AwardInfoContainer.jsx

```diff
@@ -0,0 +1,112 @@
+/**
+  * AwardInfoContainer.jsx
+  * Created by Emily Gullo 01/20/2017
+  **/
+
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import moment from 'moment';
+
+import SummaryBar from 'components/award/SummaryBar';
+import AwardInfoBar from 'components/award/AwardInfoBar';
+
+import * as awardActions from 'redux/actions/award/awardActions';
+
+const propTypes = {
+    getSelectedAward: React.PropTypes.func
+};
+
+class AwardInfoContainer extends React.Component {
```

Comment:
Once the award fetching logic is taken over by the `AwardContainer`, there may not be a need to keep this as a container; it can be downgraded to a component.